src/Components/QuestionList/QuestionViewTypes/ViewSelect/ViewSelect.tsx

```diff
@@ -0,0 +1,19 @@
+import { Form } from "react-bootstrap";
+
+interface Props {
+    setValue(value: number): void;
+}
+
+export const ViewSelect = ({ setValue }: Props) => {
+    return (
+        <Form.Select
+            onChange={(e) => {
+                const view = parseInt(e.target.value);
+                setValue(view);
+            }}
+        >
+            <option value={1}>Cards</option>
+            <option value={2}>Table</option>
```

Comment:
this should be a map of the available views not hard coded. also the views types should be an enum